docs/addons.md

```diff
@@ -322,6 +322,18 @@ spec:
       enabled: true
 ```
 
+##### Self-provisioned aws-ebs-csi-driver
```

Comment:
```suggestion
##### Self-managed aws-ebs-csi-driver
```

nit: since we use the term managed in our configs, it's probably good to be consistent in terminology.